tools/ddrgen/src/scanners/dwarf/AixDwarfParser.hpp

```diff
@@ -19,41 +19,16 @@
 #include "config.hpp"
 #include "DwarfFunctions.hpp"
 
-const string DATA_ROW ="^\\[.*\\].*m.*debug.*decl\\s+";
-const string END_OF_FILE_SECTION_ROW = ".*\\s(m)*\\s*.text\\s*(1)\\s*(unamex)\\s*(\\*\\*No Symbol\\*\\*)";
-const string START_OF_FILE_SECTION_ROW = ".*\\s*m.*debug\\s*3*\\s*FILE\\s*";
-const string FILE_INFORMATION_ROW = "\\s(a0)\\s*";
-const string NAME ("[^;:'\"]*");
-const string INTEGER ("(-)?[0-9]+");
-const string NON_NEG_INTEGER ("[0-9]+");
-const string NUM_BYTES (NON_NEG_INTEGER);
-const string NUM_BITS (NON_NEG_INTEGER);
-const string BIT_OFFSET (INTEGER);
-const string CLASS_TYPEID (NON_NEG_INTEGER);
-const string ORD_VALUE (INTEGER);
-const string CLASS_KEY ("(s|u|c)");
-const string ACCESS_SPEC ("(i|o|u)"); /* MUST ALWAYS BE THERE */
-const string ANON_SPEC ("a");
-const string BASE_CLASS_OFFSET (NON_NEG_INTEGER);
-const string VIRTUAL_ACCESS_SPEC ("(v?"+ACCESS_SPEC+"?)");
-const string OPT_BASE_SPEC (VIRTUAL_ACCESS_SPEC+BASE_CLASS_OFFSET+"\\:"+CLASS_TYPEID+",?"); /* comma if we have multiple of this */
+const string START_OF_FILE = "debug     3    FILE";
```

Comment:
As discussed: consider a more robust approach that does not require an exact amount of whitespace and column width